internal/ingress/controller/template/configmap.go

```diff
@@ -427,3 +429,22 @@ func splitAndTrimSpace(s, sep string) []string {
 
 	return values
 }
+
+func dictStrToKb(sizeStr string) int {
+	sizeMatch := dictSizeRegex.FindStringSubmatch(sizeStr)
+	if sizeMatch == nil {
+		return -1
```

Comment:
I use the convention of a negative return value implying an error here. If there is a preferred format, let me know. It just seemed most minimal and intuitive to me when making this.